pom.xml

```diff
@@ -423,6 +423,10 @@
 					<artifactId>bcprov-jdk16</artifactId>
 					<groupId>bouncycastle.bcprov-jdk16</groupId>
 				</exclusion>
+				<exclusion>
```

Comment:
This fixes build problems during deploy. otherwise conflict with multiple bouncy-castle dependencies.